### ISSUE TITLE
Warn: Spawn blocking entity has oldEnt == nullptr

### ISSUE THREAD

**Issue by illwieckz** [UnvanquishedAssets/map-parpax_src.dpkdir#3]
While running the map build from this repository I get a continuous warning message printed:

```
Warn: Spawn blocking entity has oldEnt == nullptr
```

I don't know what it means but it does not look critical and can wait for the 0.52.0 release.

**Comment by mjstahlberg** [comment]
I haven't seen this, can you give me a link to the map you built?

This seems to be a message from the server side gamelogic (so players wouldn't see it). There is an entity that blocks some spawn (could even be a spectator spawn?) which is lacking the pre-CBSE entity structure. I don't know what kind of entity this could be.

**Comment by illwieckz** [comment]
to get my latest build:

```
wget -O- http://gg.illwieckz.net/dl/unvanquished/dev/UnvanquishedAssets.get | xargs wget
```

**Comment by mjstahlberg** [comment]
Of course you have this automated somehow… :grin:

**Comment by mjstahlberg** [comment]
I'm not seeing a warning like that when testing `map-parpax_0+20181006-040426+6a18999.dpk` locally on `for-0.51.0`, commit `a001013d1`. Do you have any sgame debug cvars switched on?

**Comment by illwieckz** [comment]
For cvars, see https://github.com/illwieckz/buildme-unvanquished/blob/49e5665288a6e412f01731521d8e3c5d1ef1c1cc/Makefile#L56

The point of automating things is that it's also documenting things. ;-)

**Comment by mjstahlberg** [comment]
Oh I remember now, this is from the `sgame.spawn` logger! The relevant method is `SpawnerComponent::Think`. However, since it's a warning, it should be printed by default every 30 seconds. Even with the loglevel set to `debug`, I'm not seeing any such message. All four default spawns on Parpax seem to be operating as well. Can you check if one is not operating on your end @illwieckz?

**Comment by illwieckz** [comment]
how to do this?

**Comment by mjstahlberg** [comment]
You add a lot of bots and see if each egg and telenode spawns one. :wink:

**Comment by illwieckz** [comment]
Fixed in 276de7c3e90c4cffdf90fcabed1a4dc5584c65e6
